docs/sources/dashboards/share-dashboards-panels/_index.md

```diff
@@ -267,10 +267,10 @@ https://play.grafana.org/render/d-solo/ktMs4D6Mk?from=2024-09-03T11:55:44.442Z&t
 
 ### Share an embed
 
-You can share a panel by embedding it on another website using an iframe. Users must be signed into Grafana to view the panel.
+You can share a panel by embedding it on another website using an iframe. Users must be signed into Grafana to view the panel even if it is from a Public Dashboard in Grafana Cloud. 
```

Comment:
The combination of this update and the one at line 301 is confusing to me. Once we've clarified line 301, then I'll come back to this. But it seems like what you've added here is saying the same thing as what's in the note on line 273, no?

Reply:
You're not wrong, it's redundant. I'm just trying to make it apparent to folks that public vs private won't matter, they've still gotta be signed in due to `X-Frame-Options` to prevent displaying.